src/app/plan/components/Wizard/VolumesTable.tsx

```diff
@@ -49,46 +57,124 @@ class VolumesTable extends React.Component<any, any> {
   getTableData() {
     // Builds table data from a combination of the formik values, and the
     // persistent volumes as seen on a MigPlan object.
-    const discoveredPersistentVolumes = this.props.currentPlan.spec.persistentVolumes;
-
-    // No PVs discovered to be in use. This is normal for stateless cloud apps.
-    if(!discoveredPersistentVolumes) {
-      return [];
-    }
+    const currentPlan = this.props.plans.find(p => {
+      return p.metadata.name === this.props.values.planName;
+    });
+    if (currentPlan && currentPlan.spec.persistentVolumes) {
+      const discoveredPersistentVolumes = currentPlan.spec.persistentVolumes;
 
-    return discoveredPersistentVolumes.map(planVolume => {
-      let pvAction = 'copy'; // Default to copy
-      if(this.props.values.persistentVolumes.length !== 0) {
-        const rowVal = this.props.values.persistentVolumes.find(v => v.name === planVolume.name);
-        pvAction = rowVal.type;
+      // No PVs discovered to be in use. This is normal for stateless cloud apps.
+      if (!discoveredPersistentVolumes) {
+        return [];
       }
+      if (this.props.values.prersistentVolumes) {
```

Comment:
Some logic is broken here unless there's a consistent typo: `prersistentVolumes`

Reply:
updated this to work now. Nice catch